source/module_hamilt_lcao/module_gint/kernels/cuda/cuda_tools.cu

```diff
@@ -190,6 +190,25 @@ inline void Cuda_Mem_Wrapper<T>::copy_host_to_device_async(cudaStream_t stream,
         stream));
 }
 
+template <typename T>
+inline void Cuda_Mem_Wrapper<T>::copy_host_to_device_async(cudaStream_t stream,
```

Comment:
use &stream instead?

Reply:
Since `cudaStream_t` is a pointer type, it does not require passing by reference when used as a function argument.
![image](https://github.com/deepmodeling/abacus-develop/assets/153698752/1f7b6712-c29b-426e-a335-3ae12ca0d62a)